dev/breeze/tests/test_pytest_args_for_test_types.py

```diff
@@ -25,15 +25,18 @@
 
 # TODO(potiuk): rename to all_providers when we move all providers to the new structure
 def _all_new_providers() -> list[str]:
```

Comment:
We can do this rename now, ya?

Reply:
I prefer to do it next... It's already FAR too many changes :).